llvm/test/tools/llvm-objcopy/ELF/Inputs/srec-elf-sections-err.yaml

```diff
@@ -0,0 +1,22 @@
+!ELF
```

Comment:
This file is only used in one place. Please inline it into that test, so that the test input data can be found with the rest of the test.